monitor/README.md

```diff
@@ -27,5 +27,6 @@ This script will function as a service continually querying deployed subgraphs a
 
 Need to post errors for messages, but only in the case it is the first time an error type is detected on a deployment or if it is a reminder
 
+-Discord message for each check group
```

Comment:
This is still not fixed. The bullet points are not rendering properly

Reply:
Now fixed to Github formatting standards